### PR TITLE
Python Config

### PR DESCRIPTION
**Pull request by supertick**

### CODE REVIEW

File: src/main/java/org/myrobotlab/service/Runtime.java [hunk 1]
```diff
@@ -1476,7 +1474,7 @@ synchronized public static void unregister(String inName) {
       }
 
       if (updatedServiceLists) {
-        runtime.broadcast("publishInterfaceToPossibleServices");
+        runtime.invoke("publishInterfaceToPossibleServices");
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/Runtime.java:1477]: this isn't part of the service lifecycle publisher / listener interfaces.. if we are putting it here.. it should be defined in the interface also.

File: src/main/java/org/myrobotlab/service/Runtime.java [hunk 2]
```diff
@@ -3660,8 +3652,16 @@ static public ServiceConfig load(String name) throws IOException {
 
     ServiceConfig config = CodecUtils.readServiceConfig(filename);
     si.load(config);
+
+    if (runtime != null) {
+      runtime.invoke("publishConfigLoaded", name);
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/Runtime.java:3657]: also.. this isn't part of the service lifecycle interfaces.

File: src/main/java/org/myrobotlab/service/WebGui.java [hunk 3]
```diff
@@ -1320,4 +1322,24 @@ public static void main(String[] args) {
       log.error("main threw", e);
     }
   }
+
+  @Override
+  public void onCreated(String name) {
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/WebGui.java:1327]: shouldn't there be some implementation here?  if not, why is WebGui implementing the ServiceLifeCycleListener?

File: src/main/java/org/myrobotlab/service/Runtime.java (hunk 2 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/service/Runtime.java:3657]: this is not part of the ServiceLifeCycleListener interface and should not be

File: src/main/java/org/myrobotlab/service/WebGui.java (hunk 3 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/service/WebGui.java:1327]: webgui has to keep the ui in sync with any new service, it does so by subscribing to lifecycle events..
it only needs the onStarted

if you remember synching the availability of services for attaching has gone through a couple iterations ...
I'm not positive that WebGui does not need the onStarted events but I didn't want to keep adding changes to this pr

File: src/main/java/org/myrobotlab/service/Runtime.java (hunk 1 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/service/Runtime.java:1477]: it shouldn't be either - its currently an very optimized method that only is useful for the ui

**Pull request merged**